api/Document.json

```diff
@@ -11679,16 +11677,24 @@
           "support": {
             "chrome": [
               {
-                "version_added": "33"
+                "version_added": "33",
+                "notes": "Before Chrome 62, the <code>onvisibilitychange</code> attribute is not supported. To listen to this event in earlier versions of Chrome, use <code>document.addEventListener('visibilitychange', function() {});</code>."
               },
               {
                 "version_added": "13",
                 "prefix": "webkit"
               }
             ],
-            "chrome_android": {
-              "version_added": "33"
-            },
+            "chrome_android": [
+              {
+                "version_added": "33",
+                "notes": "Before Chrome 62, the <code>onvisibilitychange</code> attribute is not supported. To listen to this event in earlier versions of Chrome, use <code>document.addEventListener('visibilitychange', function() {});</code>."
+              },
+              {
+                "version_added": "18",
+                "prefix": "webkit"
+              }
+            ],
             "edge": {
               "version_added": "12",
               "notes": "The <code>onvisibilitychange</code> attribute was not supported until Edge 18. To listen to this event in earlier versions of Edge, use <code>document.addEventListener('visibilitychange', function() {});</code>."
```

Comment:
```suggestion
              "notes": "Before Edge 18, the <code>onvisibilitychange</code> attribute is not supported. To listen to this event in earlier versions of Edge, use <code>document.addEventListener('visibilitychange', function() {});</code>."
```

Reply:
Right, @ddbeck?

Reply:
Yes! I didn't touch the ones outside the +/- diff bounds—thanks for catching this one. If you want to change this (and others) and merge at your discretion, I'm fine with that. 👍

Reply:
@foolip Are you planning to come back to this? I was worried it slipped through the cracks, since you're not the original author